packages/mobile/src/components/track-list/TrackList.tsx

```diff
@@ -124,7 +141,9 @@ export const TrackList = ({
       isActive: false
     }) as ReactElement
 
-  return isReorderable ? (
+  return showSkeleton ? (
```

Comment:
Could we do a separate return statement instead of nested ternary? That seem easier to read